### DISCUSSION TITLE
About student registration for student developer pack

### DISCUSSION THREAD

**Discussion by Aditya84x** [community/community#82922] [GitHub Education] [answered]
I'm a student from COEP Technological University. Also I have logged in via college email ID which is godaseav23.comp@coeptech.ac.in, but when I go to register for student developer pack it selects my college as AISSMS. 

What to do?

**Comment by Talha-Hamza** [comment]
I think you should be able to select your school under the "what is the name of your school?" question.

**Reply by Aditya84x** [reply]
No, there is no option for changing it.

**Reply by SurajjBhardwaj** [reply]
are you also not able to choose another a school ?

**Reply by Aditya84x** [reply]
> are you also not able to choose another a school ? 

Means you also.

**Comment by SurajjBhardwaj** [comment]
apply one more time using the I'D card of your school.

**Reply by Aditya84x** [reply]
While registration this is the second step and I am unable to move forward from first step.

**Comment by ghostinhershell** [comment]
Hey there!  👋

Thanks for posting in the GitHub Community, @Aditya84x! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!

**Comment by shivpxl** [comment, marked as answer]
I have solved this issue by github contact support team, now you can continue to login with this - coeptech.ac.in mail and from now it will show COEP Technological University, {ShivPal Singh Coep'27 CS } 😃
